src/print.jl

```diff
@@ -324,6 +324,94 @@ function objective_function_string(print_mode, model::Model)
    end
 end
 
+"""
+    show_solution_summary([io::IO], model::Model; verbose::Bool = false)
+
+Write to `io` (or to the default output stream `stdout` if `io` is not given) a summary of the solution results.
+If `verbose` is true, additional information are written.
+"""
+function show_solution_summary(io::IO, model::Model; verbose::Bool = false)
+    print(io, "* Solver:", solver_name(model), "\n")
+    print(io, "\n")
+
+    _show_status_summary(io, model, verbose)
+    _show_candidate_solution_summary(io, model, verbose)
+    _show_work_counters_summary(io, model, verbose)
+end
```

Comment:
Add an explicit `return` at the end of this function.

Reply:
done.